src/components/TodoList/TodoList.scss

```diff
@@ -0,0 +1,3 @@
+.TodoList {
+  padding: 20px;
```

Comment:
Consider using more descriptive class names. `.TodoList` is too generic and may cause naming conflicts with other components or libraries. Consider using a more specific name like `.TaskList` or `.TodoItems`. 🤔